code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -0,0 +1,224 @@
+	dangerrating = 3
+
+/mob/living/simple_animal/hades/Life()
+	..()
+	if(world.time > lastsinPerson + sinPersonTime)
+		if(prob(fakesinPersonChance))
+			lastsinPerson = world.time
+			world << "<span class='warning'><font size=4>[pick(sinPersonsayings)]</font></span>"
+			//SUE ME
+			var/list/creepyasssounds = list('sound/effects/ghost.ogg', 'sound/effects/ghost2.ogg', 'sound/effects/Heart Beat.ogg', 'sound/effects/screech.ogg',\
+						'sound/hallucinations/behind_you1.ogg', 'sound/hallucinations/behind_you2.ogg', 'sound/hallucinations/far_noise.ogg', 'sound/hallucinations/growl1.ogg', 'sound/hallucinations/growl2.ogg',\
+						'sound/hallucinations/growl3.ogg', 'sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg', 'sound/hallucinations/i_see_you1.ogg', 'sound/hallucinations/i_see_you2.ogg',\
+						'sound/hallucinations/look_up1.ogg', 'sound/hallucinations/look_up2.ogg', 'sound/hallucinations/over_here1.ogg', 'sound/hallucinations/over_here2.ogg', 'sound/hallucinations/over_here3.ogg',\
+						'sound/hallucinations/turn_around1.ogg', 'sound/hallucinations/turn_around2.ogg', 'sound/hallucinations/veryfar_noise.ogg', 'sound/hallucinations/wail.ogg')
+			world << pick(creepyasssounds)
+		else
+			lastsinPerson = world.time
+			var/mob/living/carbon/human/sinPerson = pick(player_list)
+			if(sinPerson)
+				loc = get_turf(pick(oview(1,sinPerson)))
+				Appear()
+				var/sinPersonchoice = pick(list("Greed","Gluttony","Pride","Lust","Envy","Sloth","Wrath"))
+				switch(sinPersonchoice)
+					if("Greed")
+						src.say("Your sin, [sinPerson], is Greed.")
+						if(prob(50))
+							src.say("I will indulge your sin, [sinPerson].")
+							var/list/greed = list(/obj/item/stack/sheet/mineral/gold,/obj/item/stack/sheet/mineral/silver,/obj/item/stack/sheet/mineral/diamond)
+							for(var/i in 1 to 10)
+								var/greed_type = pick(greed)
+								new greed_type(get_turf(sinPerson))
+						else
+							src.say("Your sin will be punished, [sinPerson]!")
+							new/obj/structure/closet/statue(get_turf(sinPerson),sinPerson)
+					if("Gluttony")
+						src.say("Your sin, [sinPerson], is Gluttony.")
+						if(prob(50))
+							src.say("I will indulge your sin, [sinPerson].")
+							var/list/allTypes = list()
+							for(var/A in typesof(/obj/item/weapon/reagent_containers/food/snacks))
+								var/obj/item/weapon/reagent_containers/food/snacks/O = A
+								if(initial(O.cooked_type))
+									allTypes += A
+							for(var/i in 1 to 10)
+								var/greed_type = pick(allTypes)
+								new greed_type(get_turf(sinPerson))
+						else
+							src.say("Your sin will be punished, [sinPerson]!")
+							sinPerson.reagents.add_reagent("nutriment",1000)
+					if("Pride")
+						src.say("Your sin, [sinPerson], is Pride.")
+						if(prob(50))
+							src.say("I will indulge your sin, [sinPerson].")
+							for(var/mob/living/carbon/human/H in player_list)
+								if(H == sinPerson)
+									continue
+								ticker.mode.traitors += H.mind

```

Comment:
Is this considered part of a subsystem? I didn't think much would change in terms of the way antagonists work.